apps/dashboard/src/main/java/com/akto/listener/InitializerListener.java

```diff
@@ -676,6 +682,49 @@ public void accept(Account t) {
         }, 0, 15, TimeUnit.MINUTES);
     }
 
+    public void setUpSensitiveMapInApiInfoScheduler() {
+        int cronTime = 15;
```

Comment:
why is this defined inside the function

Reply:
i wanted to pass this crontime to the function called in this scheduler

Reply:
keep it as a constant somewhere and avoid declaring locally inside the function